resources/EPvalidator.rb

```diff
@@ -204,8 +204,8 @@ def self.run_validator(hpxml_doc)
         "Thickness" => one, # Use zero for dirt floor
         "ExposedPerimeter" => one,
         "PerimeterInsulationDepth" => one,
-        "[UnderSlabInsulationWidth | [UnderSlabInsulationSpansEntireSlab='true']]" => one,
-        "[DepthBelowGrade | [InteriorAdjacentTo!='living space' and InteriorAdjacentTo!='garage']]" => one_or_more, # DepthBelowGrade only required when InteriorAdjacentTo is 'living space' or 'garage'
+        "UnderSlabInsulationWidth | UnderSlabInsulationSpansEntireSlab='true'" => one,
```

Comment:
This doesn't look right. I tried a HPXML that has
```
            <UnderSlabInsulationWidth>2.0</UnderSlabInsulationWidth>
            <UnderSlabInsulationSpansEntireSlab>false</UnderSlabInsulationSpansEntireSlab>
```
and the EPvalidator incorrectly threw an error
`Error: C:/GitHub/OpenStudio-HPXML/tests/base-foundation-slab.xml: Expected [1] element(s) but found 2 element(s) for xpath: /HPXML/Building/BuildingDetails/Enclosure/Slabs/Slab/UnderSlabInsulationWidth | UnderSlabInsulationSpansEntireSlab='true'`

I think it needs to be
```suggestion
        "UnderSlabInsulationWidth | [UnderSlabInsulationSpansEntireSlab='true']" => one,
```

Reply:
True, I thought they're the same. Will fix it.